system-parachains/people/people-kusama/src/identity_ops.rs

```diff
@@ -0,0 +1,182 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[frame_support::pallet]
+pub mod pallet_identity_ops {
+	use crate::*;
+	use frame_support::pallet_prelude::*;
+	use frame_system::pallet_prelude::*;
+	use pallet_identity::Judgement;
+
+	type IdentityPallet = pallet_identity::Pallet<Runtime>;
```

Comment:
Why are we using the concrete `Runtime` here?

Reply:
Why not? Generic instead? This is valid only for the scope of this crate.

Reply:
I mean the pallet uses `Pallet<T>, Error<T>, etc` but still depends concretely on Runtime. Seems a bit inconsistent to me.

Reply:
I do not think it will compile without `<T>` argument for those types. For the rest where I can I use the actual type for simplicity.

Reply:
You think it worth generalizing these pallet for consistency?

Reply:
I just realised that you are using it a number of other places as well - accessing the Balances pallet and such. So, probably not worth it at this point.

Reply:
This pallet is meant to be removed by the following release. I do not see a practical reason to generalized it.